public/content/documentation/web/component/table.md

```diff
@@ -4,192 +4,248 @@ How to test a table
 
 ## Condensed
 
-### a11y - Web Accessibility Acceptance Criteria
+### #a11y - Web Accessibility Acceptance Criteria
 
 How to test a table
 
 1. Test keyboard only, then screen reader + keyboard actions
 
-   &mdash; Arrow-keys: The table scrolls into view (and with a screenreader the cells become individually readable)
+   - Arrow-keys: The table scrolls into view (and with a screenreader the cells become individually readable)
 
 2. Test mobile screenreader gestures
 
-   &mdash; Swipe: The table is browsed from cell to cell
+   - Swipe: The table is browsed from cell to cell
 
 3. Listen to screenreader output on all devices
 
-   &mdash; Name: The table has a caption or a heading to describe its purpose
+   - Name: The table has a caption or a heading to describe its purpose
+   - Role: It identifies itself as a table
+   - Group: Column headers and row headers are identified with screenreader shortcuts
 
-   &mdash; Role: It identifies itself as a table
-
-   &mdash; Group: Column headers and row headers are identified with screenreader shortcuts
-
-Full information: https://www.magentaa11y.com/#/web-criteria/component/table
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/table](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/table)
 
 ## Gherkin
 
-### a11y - Web Accessibility Acceptance Criteria
+### #a11y - Web Accessibility Acceptance Criteria
 
 How to test a table
 
-GIVEN THAT I am on a page with a table
+GIVEN THAT I am on a page with a table landmark
 
 1. Keyboard for mobile & desktop
 
-   &mdash; WHEN I use the arrow keys  
-   &mdash; I SEE the table scrolls into view (but is not focusable)
+   - WHEN I use the arrow keys I SEE the table scrolls into view (but is not focusable)
 
 2. Desktop screenreader
 
-   &mdash; WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND  
-   &mdash; I use the arrow keys
-
-   &mdash; I HEAR The table has a caption or a heading to describe its purpose  
-   &mdash; I HEAR It identifies itself as a table  
-   &mdash; I HEAR Column headers and row headers are identified with screenreader shortcuts
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver)
+   - AND use the arrow keys
+      - I HEAR The table has a caption or a heading to describe its purpose
```

Comment:
All first letters after "I HEAR in steps 2 and 3 of Gherkin are capitalized and don't need to be.